src/main/java/org/testng/TestNG.java

```diff
@@ -1407,6 +1420,7 @@ protected void configure(CommandLineArgs cla) {
     setMixed(cla.mixed);
     setSkipFailedInvocationCounts(cla.skipFailedInvocationCounts);
     toggleFailureIfAllTestsWereSkipped(cla.failIfAllTestsSkipped);
+    setListenersToSkipFromBeingWiredInViaServiceLoaders(cla.spiListenersToSkip.split(","));
```

Comment:
Here I am trying to accept a single value as a comma separated value, and then splitting them inside. I am not sure if JCommander would do that.

I don't want to do something like this 

```
java Main -host host1 -verbose -host host2
```

because I am not sure if we would be able to support this via surefire plugin executions, because there everything is given to TestNG as a Map and the values will get overridden.

Reply:
In that case, you have the other solution: http://jcommander.org/#list-value

Reply:
@juherr - Yes I could very well use that, but then again, I would have to apply the "Split a csv string to a list" logic when I am trying to read its value from a `Map` that gets passed to me via surefire plugin. So my point was, why do I need to apply the same logic in two places, when I could easily put it in one place that reads the value from this parameter. See [here](https://github.com/cbeust/testng/pull/2261/files#diff-c89ea27c996c668601a33f887b5675f2R1581)

I agree with your point of leveraging what JCommander gives to us, but in this case, TestNG can be initialized either via command line arguments (which is where JCommander will help ) or via a Map.

I feel that the current approach of me manually translating the csv into a list internally is probably less verbose and solves the purpose than leveraging the list parsing capabilities of JCommander.

Reply:
ok